web/atlas.json

```diff
@@ -5231,7 +5213,6 @@
 {"id": "txbnyg", "submitted_by": "StarbirbZ", "name": "StarbirbZ", "description": "Bird and heart made by StarbirbZ, defended by the ROSEWOOD community", "website": "https://twitter.com/StarbirbZ", "subreddit": "", "center": [1870.5, 51.5], "path": [[1865.5, 49.5], [1865.5, 54.5], [1872.5, 54.5], [1876.5, 52.5], [1875.5, 49.5]]},
 {"id": "txbnpb", "submitted_by": "Quandtrium", "name": "Pomu Rainpuff", "description": "Pomu Rainpuff (ぽむ れいんぱふ) is an English female VTubter from nijisanji en, build by the /r/place Vtubers discord", "website": "", "subreddit": "", "center": [1352.5, 1051.5], "path": [[1345.5, 1044.5], [1358.5, 1044.5], [1358.5, 1057.5], [1357.5, 1057.5], [1357.5, 1059.5], [1349.5, 1059.5], [1349.5, 1058.5], [1346.5, 1058.5], [1346.5, 1057.5]]},
 {"id": "txbnj7", "submitted_by": "Wrath_FMA", "name": "CatTail|Plants vs. Zombies", "description": "CatTail, The evolved form of Lily pad from Plants vs. Zombies", "website": "", "subreddit": "/r/PlantsVSZombies", "center": [1465.5, 987.5], "path": [[1452.5, 975.5], [1452.5, 998.5], [1478.5, 998.5], [1478.5, 975.5], [1452.5, 975.5]]},
-{"id": "txbn0e", "submitted_by": "ProfessionalFish8505", "name": "Octoling Octopus", "description": "Pixel art based on Octolings from Splatoon 2. Specifically, this is the octopus form they can turn into to swim through ink or superjump.", "website": "", "subreddit": "/r/splatoon", "center": [1127.5, 1856.5], "path": [[1119.5, 1849.5], [1119.5, 1863.5], [1135.5, 1863.5], [1135.5, 1849.5], [1119.5, 1849.5], [1119.5, 1849.5]]},

```

Comment:
This has info that the other description doesn't. Merge the two descriptions to preserve that info.

Reply:
Got it, thank you for pointing it out. Resolved in my last commit: https://github.com/placeAtlas/atlas/pull/1146/commits/5ba9a5ebf4d7ca121e77159b0eaa82a6e4b0981e